### ISSUE TITLE
[Suggestion] Credits listed in-game.

### ISSUE THREAD

**Issue by SheepishShepherd** [tModLoader/tModLoader#687]
Obviously if this were to exist, there would need to be a way for it to automatically update itself. I'm not sure how to do that myself, but if there is a way I would like to see this.

**Comment by Jofairden** [comment]
Yeah we were working on this a while ago, I'm not sure what happened to it. @JavidPack  ?

**Comment by Plantterror** [comment]
If this gets done, might I also suggest a "support" page nearby linking to some useful websites (github wiki, discord, forum page, etc) maybe similar to how @dradonhunter11 has done in 64 bit currently. It probably isn't a good idea to merge the Patreon Page with the support page, however.

**Comment by GoodPro712** [comment]
On the main menu could there be a button above the tModLoader patron link in the bottom left corner named Support and credits. When pressed it opens a menu that displays the following text.
Credits Official Website Official Discord Server Github Releases Wiki
When pressing on Official Website, Official Discord Server, Github, Releases, and Wiki, it will Process.Start the links respectively
When pressing on Credits, it will open a menu that will display the 4 API developers, Blushiemagic, Jopojelly, Chicken Bones, Jofairden. It would then display Thank you to all of our contributors There will also be another tab under the contributor thanking which would be saying Thanks to all of our Skeletron and above tier patrons ($10). When pressed will open a UIInfoMessage and list every skeletron and above tier patron as the patron says "Will be listed in the tModLoader credits"
